$OpenBSD: patch-help_LINGUAS,v 1.1 2019/11/02 09:38:54 ajacoutot Exp $

Running custom install script '/usr/local/bin/meson --internal yelphelper install --subdir=help --id=deja-dup --installdir=share/help --sources=backup-auto.page@@backup-first.page@@contribute.page@@credits.page@@index.page@@license.page@@prefs.page@@restore-full.page@@restore-lost.page@@restore-revert.page@@restore-worst-case.page@@support.page --symlinks=true'
Traceback (most recent call last):
  File "/usr/local/bin/itstool", line 1598, in <module>
    doc.merge_translations(translations, opts.lang, strict=opts.strict)
  File "/usr/local/bin/itstool", line 962, in merge_translations
    self.merge_translations(translations, language, node=child, strict=strict)
  File "/usr/local/bin/itstool", line 962, in merge_translations
    self.merge_translations(translations, language, node=child, strict=strict)
  File "/usr/local/bin/itstool", line 964, in merge_translations
    newnode = self.get_translated(node, translations, strict=strict, lang=language)
  File "/usr/local/bin/itstool", line 1065, in get_translated
    msgstr.encode('utf-8')))
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 18: ordinal not in range(128)

Index: help/LINGUAS
--- help/LINGUAS.orig
+++ help/LINGUAS
@@ -35,7 +35,6 @@ ru
 sk
 sl
 sr
-sv
 th
 tr
 vi
